packages/design-system/src/styles/_reset.scss

```diff
@@ -0,0 +1,248 @@
+/* Core root defaults */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html:focus-within {
+  scroll-behavior: smooth;
+}
+
+// Remove all animations, transitions and smooth scroll
+// for people that prefer not to see them
+@media (prefers-reduced-motion: reduce) {
+  html:focus-within {
+    scroll-behavior: auto;
+  }
+
+  *,
+  *::before,
+  *::after {
+    animation-duration: 0.01ms !important;
+    animation-iteration-count: 1 !important;
+    scroll-behavior: auto !important;
+    transition-duration: 0.01ms !important;
+  }
+}
+
+/* Core body defaults */
+
+body {
+  background-color: var(--color-background);
+  color: var(--color-base);
+  font-family: var(--typography-body__font-family);
+  font-size: var(--font-size-base);
+  line-height: var(--font-line-height-base);
+  margin: 0;
+  min-height: 100vh;
+  text-rendering: optimizeSpeed;
+}
+
+// This removes focus rings for non-interactive elements,
+// i.e. when activating a skip link will then send focus
+// to a main or a heading element.
+[tabindex='-1']:focus {
+  outline: 0;
```

Comment:
There was discussion between @pwolfert and @SparrowSpit about whether this rule would be adequate for what teams used it for. Dan pointed out a caveat might be if a heading were a link, but I still think this would work - mainly because the link nested inside the heading would still receive an outline as it is interactive:

```html
<h2 tabindex="-1">
    <a href="">Click this heading to go nowhere</a>
</h2>
```

A quick codepen test showed the link still receives a focus ring despite the heading with a -1 tabindex having this style.

Reply:
Agree 👍